### DISCUSSION TITLE
Online Funding website for GitHub README.md, and gmail

### DISCUSSION THREAD

**Discussion by IMGROOT2** [For-0/vocabustudy#186] [Ideas]
So, I updated https://github.com/For-0/.github/blob/main/profile/README.md, and I think we should add something like Patreon, buymeacoffee, etc., in case somebody comes across it and decides to donate, lol

also, since we might be doing more online/related things with ForZero, I think having a team Gmail account would be helpful. what do you think? @grimsteel, if you're still using your personal email for firebase, we can move it over to a team gmail, so your storage isn't used up

it's 3:05 I need to sleep LOL. finally, @grimsteel, please look over my pull request; I'm not sure if I did everything correctly. I accidentally submitted a merge request for the emulator data, but I reverted it. IDK if it went through.

**Comment by grimsteel** [comment]
> having a team Gmail account

I have vocabustudy@googlegroups.com

> we should add something like Patreon, buymeacoffee, etc., in case somebody comes across it and decides to donate, lol

Sure do what you want

**Reply by IMGROOT2** [reply]
I'm not sure, honestly, your choice. which one has the lowest fee?

**Reply by grimsteel** [reply]
Locker donation box lol
Looks like gofundme has the lowest (2.5%?) fee

**Reply by grimsteel** [reply]
> I have [vocabustudy@googlegroups.com](mailto:vocabustudy@googlegroups.com)

I changed it to forzero@googlegroups.com

**Reply by grimsteel** [reply]
> if you're still using your personal email for firebase

I am, but I don't think it should really affect storage

**Reply by grimsteel** [reply]
The google group is just for things that ask me what a public email should be

**Reply by IMGROOT2** [reply]
>Looks like gofundme has the lowest (2.5%?) fee

I was thinking about that, but it's not a good option. GoFundMe is for people who are going through a hard time, etc. Kickstarter might be a much better idea since it's made for our kind of stuff.

> I changed it to [forzero@googlegroups.com](mailto:forzero@googlegroups.com)
> The google group is just for things that ask me what a public email should be

Great, but how does a google groups email work? Does it work just like Gmail? I thought it was for email lists. And, if/when we get forzero.org (hopefully sometime in the future), we can get a custom domain, like master@forzero.org.

> I am, but I don't think it should really affect storage

Great nvm then

**Comment by grimsteel** [comment]
> it's 3:05 I need to sleep LOL

I'm curious: what time did you wake up today?

I slept at like 1 last night:
![image](https://user-images.githubusercontent.com/115331356/197595324-667d150b-bb36-4c89-8d30-9991cae3f461.png)
I feel like the best programs get written around 12 AM lol

**Reply by IMGROOT2** [reply]
> I'm curious: what time did you wake up today?

It was so weird: I woke up at 10:05 or something, but after only 6 hours of sleep, I felt so... normal. I have no idea what happened, but I feel like I got 9 hours of sleep.

> I feel like the best programs get written around 12 AM lol

Actually though! It's crazy but very true

**Comment by grimsteel** [comment]
> but after only 6 hours of sleep, I felt so... normal. I have no idea what happened, but I feel like I got 9 hours of sleep.

Same!!

**Reply by grimsteel** [reply]
I think it's because knowing that you did something cool the previous day gives you peace of mind while sleeping

Like if I slept for 6 hours during the week, I would be so tired the next morning

**Reply by IMGROOT2** [reply]
Agreed
